packages/TorneloScoresheet/src/components/Pin/Pin.tsx

```diff
@@ -19,27 +25,32 @@ const Pin = () => {
   });
 
   return (
-    <SafeAreaView style={styles.root}>
-      <CodeField
-        ref={ref}
-        {...props}
-        // Use `caretHidden={false}` when users can't paste a text value, because context menu doesn't appear
-        value={value}
-        onChangeText={setValue}
-        cellCount={CELL_COUNT}
-        rootStyle={styles.codeFieldRoot}
-        keyboardType="number-pad"
-        textContentType="oneTimeCode"
-        renderCell={({ index, symbol, isFocused }) => (
-          <Text
-            key={index}
-            style={[styles.cell, isFocused && styles.focusCell]}
-            onLayout={getCellOnLayoutHandler(index)}>
-            {symbol || (isFocused ? <Cursor /> : null)}
-          </Text>
-        )}
-      />
-    </SafeAreaView>
+    <View>
+      <SafeAreaView style={styles.root}>
+        <CodeField
+          ref={ref}
+          {...props}
+          // Use `caretHidden={false}` when users can't paste a text value, because context menu doesn't appear
+          value={value}
+          onChangeText={setValue}
+          cellCount={CELL_COUNT}
+          rootStyle={styles.codeFieldRoot}
+          keyboardType="number-pad"
+          textContentType="oneTimeCode"
+          renderCell={({ index, symbol, isFocused }) => (
+            <Text
+              key={index}
+              style={[styles.cell, isFocused && styles.focusCell]}
+              onLayout={getCellOnLayoutHandler(index)}>
+              {symbol || (isFocused ? <Cursor /> : null)}
+            </Text>
+          )}
+        />
+      </SafeAreaView>
+      <View style={styles.verifyButtonArea}>
+        <PrimaryButton onPress={() => onPress()} label="Verify" />
```

Comment:
We should pass the value of the pin into this function no?
Also you can just write `onPress={onPress}` , no need to declare another function

Reply:
If we do onPress={onPress} we get an error. We were holding off on passing the value of the pin anywhere until we talk to David about how they want it stored because we assume it will need to be encrypted or something which is outside the scope of this ticket.